packages/pluggableWidgets/app-events-native/src/AppEvents.tsx

```diff
@@ -95,8 +96,107 @@ export class AppEvents extends Component<Props> {
 
         this.isConnected = nextState.isConnected;
     }
+
+    private triggerOnUnloadEvent(): void {
+        if (this.props.onUnloadAction?.canExecute) {
+            executeAction(this.props.onUnloadAction);
+        }
+    }
+
+    private triggerOnLoadEvent(): void {
+        if (!this.onLoadTriggered && this.props.onLoadAction?.canExecute) {
+            this.onLoadTriggered = true;
```

Comment:
What is this `onLoadTriggered` flag for? Surely we are not expecting `componentDidMount` to be called multiple times? (I know the flag was already there, but the asymmetry with `triggerOnUnloadEvent` is a lot more obvious now.)

Reply:
Forgotten code afaik